src/bump/bump.ts

```diff
@@ -0,0 +1,188 @@
+/**
```

Comment:
NOTE: This file contains the shared functions between the SdkVer and SemVer bump strategies, which are now kept in their own seperate files. Unfortunately Git is not helping here 👎 

I am inclined to drop this PR altogether due to this....